lib/mongoose.js

```diff
@@ -35,7 +35,7 @@ module.exports = function FilterablePlugin(Filterable, schema, options) {
             return [
                 field,
                 {
-                    type: _.contains(VALID_TYPES, schemaField.type)? schemaField.type : null,
+                    type: _.contains(VALID_TYPES, schemaField.instance)? schemaField.type : null,
```

Comment:
The problem is that it'll not support `mongoose<4.0.0` (for example I'm using 3.8.x since the 4.x.x stills has some issues).